samples/batch/js/node/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "visual-analyzer",
+  "version": "0.1.0",
+  "private": true,
+  "homepage": "/build",
+  "dependencies": {
+    "@azure/cognitiveservices-computervision": "^7.1.0",
```

Comment:
is this file necessary? some of the dependencies are unrelated. Also please avoid local paths.